### DISCUSSION TITLE
Java/Spring/Redis onboarded to Skywalking 8.9.1 seeing an increase in ms for redis service

### DISCUSSION THREAD

**Discussion by lstephe** [apache/skywalking#9542] [General]
A recent consumer onboarded to Skywalking version 8.9.1 and has observed an increase in (~20)ms for redis service with Skywalking.  Below are the versions and I have the following questions:

Java : 1.8
Spring boot: 2.5.8
Cassandra: 3.7.2
Spring Redis: 2.5.0

1.  Are the above versions compatible/supported in Skywalking 8.9.1?
2. Have other customers on Skywalking 8.9.1 observed an increase in ms for services with the above versions?
3. Any recommendation on how to resolve the increase in ms for above versions?

**Comment by wu-sheng** [comment]
I don't think 8.9.1 is an agent version. Backend has nothing relative to your service.
Please make sure your questions accurate.

**Reply by lstephe** [reply]
The service was onboarded in GCP.  Architecture we are using skywalking satellite,  a mutation webhook which will run along side the app pod,  kafka topics and elasticsearch on the backend, the onboarding for a consumer is adding pod annotations and adding label to namespace then service is visible in the Skywalking UI,  When this was completed for the above service, they observed an increase in response time ms (~20ms) for redis service.  Below are the app versions, trying to determine why such an increase in ms for this service all other services don't have any issues.
Java : 1.8
Spring boot: 2.5.8
Cassandra: 3.7.2
Spring Redis: 2.5.0

I hope above is more clear.  Please offer any suggestion as to why increase in ms.

**Reply by wu-sheng** [reply]
You could check the collected traces, about which spans cost the 20ms.
Usually you may open some options to collect parameters? Serialization costs most time.

**Reply by lstephe** [reply]
Can you please elaborate on the options to collect parameters and serialization?  Has any other customers experienced something similar?
